tests/unittest.py

```diff
@@ -129,3 +136,66 @@ def DEBUG(target):
     Can apply to either a TestCase or an individual test method."""
     target.loglevel = logging.DEBUG
     return target
+
+
+class HomeserverTestCase(TestCase):
+
+    servlets = []
+    hijack_auth = True
+
+    def setUp(self):
+
+        self.reactor, self.clock = get_clock()
+        self._hs_args = {"clock": self.clock, "reactor": self.reactor}
+        self.hs = self.make_homeserver(self.reactor, self.clock)
```

Comment:
we've been documenting abstract methods with `abc` elsewhere:

```
    @abc.abstractmethod
    def make_homeserver(self, reactor, clock):
        """some docstring"""
        raise NotImplementedError()
```

could we do that here too?

Reply:
`abc.abstractmethod` requires the class is an ABC, which this isn't. (Also, abcs are a bad idea)